addons/assigngear/functions/fnc_testGear.sqf

```diff
@@ -205,10 +211,18 @@ private _fncTestUnit = {
             };
         } forEach _magsAndItems;
 
-        //Mag check
+        // Mag check
         if (count _primaryWeapon > 0) then {
             private _weaponMags = [_primaryWeapon select 0] call CBA_fnc_compatibleMagazines;
```

Comment:
Known issue, `CBA_fnc_compatibleMagazines` does not support UGLs or other magazines on primary weapons so will throw a warning for correctly fitting secondary primary weapon mags. 

E.g. `arifle_MX_GL_F` will complain about `1Rnd_HE_Grenade_shell` even though it fits.

Reply:
You can have it search all muzzles, https://cbateam.github.io/CBA_A3/docs/files/common/fnc_compatibleMagazines-sqf.html

Reply:
🧠

Reply:
I'm not sure if you checked the link, but you can do `[_weapon, true] call CBA_fnc_compatibleMagazines` to have it search all muzzles, instead of whatever you just did.

Reply:
I'm so blind.

Reply:
I didn't even read that, I just saw:
```
_mags = [configFile >> "CfgWeapons" >> _rifle >> _glMuzzle] call CBA_fnc_compatibleMagazines
```
😂